### DISCUSSION TITLE
Possible for Click-to-Slide-in Nav Menu using pure TailwindCSS (no JS)

### DISCUSSION THREAD

**Discussion by tian-yan** [tailwindlabs/tailwindcss#4546] [Help] [answered]
I am trying to reproduce the slide-in sidebar as shown in below URL,
[Hide the Sidebar](https://css-tricks.com/the-checkbox-hack/#hide-the-sidebar)

Above example used **Checkbox Hack** to achieve **Click-to-slide-in**.  
Based on my research, I did not see any TailwindCSS examples which produced similar effect without using other dependencies such as JavaScripts. Is that possible using pure-TailwindCSS ? I am just trying to minimize JS usage whenever possible for best compatibility.

**Comment by jenstornell** [comment, marked as answer]
It's not possible as far as I know. I use `@apply` in `tailwind.css` to get around it with custom CSS.
